src/metrics/metrics.py

```diff
@@ -1,6 +1,8 @@
 from prometheus_client.metrics import Counter, Gauge, Histogram
 
-PREFIX = 'depositor_bot'
+from variables import PROMETHEUS_PREFIX
+
+METRICS_PREFIX = PROMETHEUS_PREFIX + '_bot'
```

Comment:
This can be confusing, let's remove this prefix modification

Reply:
Done